test/clj/quil/snippet.clj

```diff
@@ -14,9 +16,42 @@
 (def tests-in-set 50)
 (def current-test (atom -1))
 
+(defn skip-automated-compare? [snippet-name]
+  (contains? #{"current-frame-rate-target-frame-rate"
+               "noise"
+               "noise-detail"
+               "random-2d"
+               "random-3d"
+               "random"
+               "random-gaussian"
+               "resize-sketch"
+               "save"
+               "time-and-date"} snippet-name))
+
+(defn- compare-images [expected actual difference]
+  (let [{:keys [err]} (sh/sh "compare" "-metric" "mae" expected actual difference)
+        result        (second (re-find #"\((.*)\)" err))]
+    (edn/read-string result)))
+
+(defn- replace-suffix [file-name suffix]
+  (string/replace file-name #"(\w+)\.(\w+)$" (str suffix ".$2")))
+
+(defn assert-unchanged-snippet-output [name]
+  (when-not (skip-automated-compare? name)
+    (let [n          (str "snippet-snapshots/clj/" name)
+          _          (q/save (str "resources/" n "-actual.png"))
```

Comment:
Can you add -actual.png into .gitignore so that they don't pollute git status when running test? Same for difference.

Reply:
done